0002-1_task/app.js

```diff
@@ -0,0 +1,51 @@
+const btn = document.getElementById('btn');
+const target = document.getElementById('task')
+const todolist = []
+
+btn.addEventListener('click', function(e) {
```

Comment:
引数が使用されていないようですので削除しましょう。
また、コールバックはアロー関数を使用すると良いでしょう。
```suggestion
btn.addEventListener('click', () => {
```